tests/gpt2/test_modeling_tf_gpt2.py

```diff
@@ -294,21 +294,6 @@ def create_and_check_gpt2_lm_head(self, config, input_ids, input_mask, head_mask
         result = model(inputs)
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
-    def create_and_check_gpt2_xla_generate(self, config, input_ids, *args):
```

Comment:
Not super happy with deleting the whole test. Could we add a skipif statement for now and correct it later?